src/index.html

```diff
@@ -5,8 +5,350 @@
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <nav class="nav header__nav">
+        <a href="#">
+          <img
+           src="images/logo.png"
+           alt="logo"
+           class="nav__homepage"
+          >
+        </a>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link nav__link--is-active is-active">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="products" data-qa="card">
+        <div class="product-cards" data-qa="card">
+          <div class="product-cards__items">
+            <img
+             src="images/imac.jpeg"
+             alt="Apple A1419"
+             class="product-cards__items-image"
+            >
+            <h1 class="product-cards__text">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </h1>
+            <p class="product-cards__code">Код товара: 195434</p>
+            <div class="product-cards__rate rate">
+              <div class="stars stars--4">
+                <span class="stars__image"></span>
+                <span class="stars__image"></span>
+                <span class="stars__image"></span>
+                <span class="stars__image"></span>
+                <span class="stars__image"></span>
+              </div>
+              <div class="rate__text">Отзывов: 5</div>
+            </div>
+            <div class="price product-cards__price">
+              <p class="price__text">Цена:</p>
+              <h2 class="price__value">69 999 грн</h2>
+            </div>
+            <a
+             href="#"
+             class="product-cards__button"
+             data-qa="card-hover"
+            >
+              КУПИТЬ
+            </a>
+          </div>
+        </div>
+      </div>
+      </div>
+      <div class="products">
+        <div class="product-cards" data-qa="card">
```

Comment:
Remove `data-qa="card"` attribute from all cards but the first one. This should fix number of tests.